ukrainian/story/eat.md

```diff
@@ -0,0 +1,3 @@
+﻿Ваш живіт вже переповнений зефіром і раптом вам захотілось
+[Зателефонувати своєму приятелеві](story/phone.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.